extensions/WSLExtension/Contracts/IProcessCreator.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using WSLExtension.Models;
+
+namespace WSLExtension.Contracts;
+
+/// <summary>
+/// Abstraction used to create processes throughout the WSL extension.
+/// </summary>
+public interface IProcessCreator
+{
+    /// <summary>
+    /// Creates and starts a new process that opens in a new Window. Note:
+    /// Execution does not block on Starting the process.

```

Comment:
What does "Execution does not block"?  Does this mean calling code won't wait for the process to be created?

Reply:
Updated the comment to be more clear, but the intent was to say that the method doesn't wait for the started process to exit before returning